test/integration/targets/bigip_device_info/tasks/issue-01757.yaml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Issue 01757 - Select all facts, single keyword
+  f5networks.f5_modules.bigip_device_info:
+    gather_subset:
+      - client-ssl-profiles
+  register: bigip_facts
+
+- name: "Display BigIP F5 Client SSL Profiles"
+  debug:
+    msg: "{{ bigip_facts }}"
```

Comment:
this is not how you should check if the right data was grabbed, use asserts and jmespath filters to find out if the data was correct, for consistency you can create resources to check for.

Reply:
the first element will always be the clientssl default profile, which can't be removed from the BIGIP, so I'll just check against that.

Reply:
if the defaul data changes between versions it might fail its better to have something uniform across versions

Reply:
Hasn't changed in 15 years.

Reply:
Fair enough, I'll add a create a resource and check against it.

Reply:
Added the clientssl profile creation and verification.